### PR TITLE
New wallet settings layout

### LINKED ISSUE #213: update wallet settings layout
use figma!

Image

### PR DESCRIPTION
**Pull request by Vanvians**
Closes #213 
Implements new layout for wallet settings page
Screenshot 2023-11-01 205355

### CODE REVIEW

File: ui/page/root/wallet_settings_page.go [hunk 1]
```diff
@@ -191,101 +187,167 @@ func (pg *WalletSettingsPage) layoutMobile(gtx C, body layout.Widget) D {
...
+		})
 	}
 }
 
 func (pg *WalletSettingsPage) account() layout.Widget {
-	dim := func(gtx C) D {
-		return pg.accountsList.Layout(gtx, len(pg.accounts), func(gtx C, a int) D {
-			return pg.subSection(gtx, pg.accounts[a].Name, pg.Theme.Icons.ChevronRight.Layout24dp)
-		})
-	}
```
Comment (itswisdomagain) [inline ui/page/root/wallet_settings_page.go:245]: Keep this please (in all the methods where `dims` was defined). Makes your changes easier to review. Also makes the code easier to read when there is fewer nesting.

File: ui/page/root/wallet_settings_page.go [hunk 2]
```diff
@@ -191,101 +187,167 @@ func (pg *WalletSettingsPage) layoutMobile(gtx C, body layout.Widget) D {
...
+			return cryptomaterial.LinearLayout{
+				Orientation: layout.Vertical,
+				Width:       cryptomaterial.MatchParent,
+				Height:      cryptomaterial.WrapContent,
+				Background:  pg.Theme.Color.Surface,
+				Direction:   layout.Center,
+				Alignment:   layout.Middle,
+				Border:      cryptomaterial.Border{Radius: cryptomaterial.Radius(14)},
+				Padding:     layout.UniformInset(values.MarginPadding24),
+			}.Layout(gtx,
```
Comment (itswisdomagain) [inline ui/page/root/wallet_settings_page.go:261]: This is the key change for this PR. This code should be in `pg.pageSections`.

File: ui/page/root/wallet_settings_page.go [hunk 3]
```diff
@@ -297,9 +359,10 @@ func (pg *WalletSettingsPage) pageSections(gtx C, title string, body layout.Widg
 				layout.Rigid(func(gtx C) D {
 					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
 						layout.Rigid(func(gtx C) D {
-							txt := pg.Theme.Label(values.TextSize14, title)
+							txt := pg.Theme.Label(values.TextSize20, title)
 							txt.Color = pg.Theme.Color.GrayText2
-							return txt.Layout(gtx)
+							txt.Font.Weight = font.Bold
```
Comment (itswisdomagain) [inline ui/page/root/wallet_settings_page.go:364]: Figma uses font weight 600. That is semi-bold.

File: ui/page/root/wallet_settings_page.go [hunk 4]
```diff
@@ -297,9 +359,10 @@ func (pg *WalletSettingsPage) pageSections(gtx C, title string, body layout.Widg
...
+							return layout.Inset{Bottom: values.MarginPadding15}.Layout(gtx, txt.Layout)
```
Comment (itswisdomagain) [inline ui/page/root/wallet_settings_page.go:365]: Space between the title and the next widget is 24px on figma. It's 26 in some places, but most places have a spacing of 24px, so let's go with 24px.

File: ui/page/root/wallet_settings_page.go [hunk 5]
```diff
@@ -349,7 +406,7 @@ func (pg *WalletSettingsPage) sectionDimension(gtx C, clickable *cryptomaterial.
 			textLabel.Color = pg.Theme.Color.Danger
 		}
 		return layout.Inset{
-			Bottom: values.MarginPadding20,
+			Bottom: values.MarginPadding15,
```
Comment (itswisdomagain) [inline ui/page/root/wallet_settings_page.go:409]: 24px on figma.

**Review by itswisdomagain — DISMISSED** [review]

**Comment by dreacot** [comment]
figma

Screenshot from 2023-11-08 11-57-23

this PR

Screenshot from 2023-11-08 11-57-48


the top, bottom, left and right paddings look off

**Comment by Vanvians** [comment]
Updated screenshot
Screenshot 2023-11-22 212831

**Review by dreacot — APPROVED** [review]

**Pull request merged**